test/src/tests-native-sdk.js

```diff
@@ -1323,71 +1105,6 @@ describe('native-sdk methods', function() {
                 done();
             });
 
-            it('should send an event with a product list when calling logPurchase', function(done) {
```

Comment:
I think this test should remain also.  logPurchase is technically deprecated too, but still works and removing it isn't part of this PR.  you can probably remove lines 1340-1349, but the rest of the test should remain. The crux of this test is for logPurchase and it happens to have some cart.add in it, but all the other non cart.add logic should remain for now